eva-accession-pipeline/src/test/resources/accession-pipeline-test.properties

```diff
@@ -3,3 +3,13 @@ accessioning.variant.blockSize=1000
 accessioning.variant.categoryId=test-ss
 
 accessioning.monotonic.init.test-ss=10000000000
+
+parameters.assemblyAccession=assembly
+parameters.taxonomyAccession=1111
+parameters.projectAccession=project
+parameters.chunkSize=100
```

Comment:
It would make sense to merge `chunkSize` and `blockSize` in a single parameter. If we don't do that in this ticket a comment should be added to the application runner JIRA ticket.